episodes/infrastructure.md

```diff
@@ -499,6 +499,46 @@ Using this approach, we can build up our lesson one episode at a time.
 We have now learned everything we need to be able to use The Carpentries Workbench,
 and our focus can return to the process of developing a new lesson.
 
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: spoiler
+
+## Troubleshooting the Lesson Build Process
+
+Sometimes, formatting errors and typos in the files of your repository can
+cause the process that builds your lesson website to fail.
```

Comment:
```suggestion
cause the process that builds your lesson website to fail. You will likely first notice the failure to build the lesson if the lesson website is not showing the changes you made. You may also receive an email from GitHub about the build failure if that is your preferred way of receiving notifications (but it may take you some time to realise this). 
```